### PR TITLE
Add manual configuration for Analytics

### PR DESCRIPTION
**Pull request by jpignata**
*Issue #, if available:* n/a

*Description of changes:* Users can use existing Amazon Pinpoint resources with the Analytics category but this was not documented. This change adds a page for iOS and Android that outlines that minimal configuration for Analytics to use existing resources.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/lib/analytics/configure.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+title: Configure existing resources
+description: Configure AWS Amplify to use existing AWS resources by pointing to them in your configuration.
+---
+
+<inline-fragment platform="android" src="~/lib/analytics/fragments/configure.md"></inline-fragment>
+<inline-fragment platform="ios" src="~/lib/analytics/fragments/configure.md"></inline-fragment>
```
Comment (jamesonwilliams) [inline docs/lib/analytics/configure.md:7]:
```suggestion
<inline-fragment platform="android" src="~/lib/analytics/fragments/configure.md" />
<inline-fragment platform="ios" src="~/lib/analytics/fragments/configure.md" />
```
Any reason these wouldn't work with the single tag?

File: docs/lib/analytics/fragments/configure.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+Existing Amazon Pinpoint resources can be used with AWS Amplify through the `awsconfiguration.json` file.
```
Comment (jamesonwilliams) [inline docs/lib/analytics/fragments/configure.md:1]:
```suggestion
Existing Amazon Pinpoint resources can be used with AWS Amplify through by including an `awsconfiguration.json` file. Note that this is ___not___ the `amplifyconfiguration.json` file.
```

File: docs/lib/analytics/configure.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+---
+title: Configure existing resources
```
Comment (brene) [inline docs/lib/analytics/configure.md:2]: Should we be more explicit here and just say "Use existing AWS resources"

File: docs/lib/analytics/configure.md [hunk 4]
```diff
@@ -0,0 +1,7 @@
...
+description: Configure AWS Amplify to use existing AWS resources by pointing to them in your configuration.
```
Comment (brene) [inline docs/lib/analytics/configure.md:3]: nit: "pointing" => "referencing"

File: docs/lib/analytics/fragments/configure.md (hunk 2 above)
Comment (brene) [inline docs/lib/analytics/fragments/configure.md:1]: is it awsconfiguration.json or amplifyconfiguration.json?

Comment (brene) [reply docs/lib/analytics/fragments/configure.md:1]: We should probably also provide the corresponding content here: https://docs.amplify.aws/cli/usage/files#amplifyconfigurationjson-file

File: docs/lib/analytics/menu.json [hunk 5]
```diff
@@ -7,6 +7,7 @@
     "identifyuser",
     "streaming",
     "storing",
-    "personalize"
+    "personalize",
+    "configure"
```
Comment (brene) [inline docs/lib/analytics/menu.json:11]: "existingresource" ?

File: docs/lib/analytics/configure.md (hunk 4 above)
Comment (mohitsriv) [inline docs/lib/analytics/configure.md:3]: Configure the Amplify Libraries to use...

File: docs/lib/analytics/configure.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (mohitsriv) [reply docs/lib/analytics/configure.md:2]: Use with existing resources?

Comment (jpignata) [reply docs/lib/analytics/configure.md:2]: That's better!

File: docs/lib/analytics/fragments/configure.md (hunk 2 above)
Comment (jpignata) [reply docs/lib/analytics/fragments/configure.md:1]: What do you mean by "corresponding content?" And yes, that's my timebox showing ;) Thanks!

File: docs/lib/analytics/menu.json (hunk 5 above)
Comment (jpignata) [reply docs/lib/analytics/menu.json:11]: This isn't reader-facing. I guess the goal here is a name that will be clear to a writer at a glance. Let's use "existingresources."

File: docs/lib/analytics/configure.md (hunk 1 above)
Comment (jpignata) [reply docs/lib/analytics/configure.md:7]: Hm, not sure, I copied-pasted. I'll try it.

File: docs/lib/analytics/fragments/configure.md (hunk 2 above)
Comment (jpignata) [reply docs/lib/analytics/fragments/configure.md:1]: Actually, it is the `amplifyconfiguration.json` file. I made a typo.

File: docs/lib/analytics/menu.json (hunk 5 above)
Comment (jpignata) [reply docs/lib/analytics/menu.json:11]: Went with existing-resources.

**Comment by jpignata** [comment]
@brene @jamesonwilliams @mohitsriv Please take another look when convenient. Thank you!

File: docs/lib/analytics/existing-resources.md [hunk 6]
```diff
@@ -0,0 +1,7 @@
+---
+title: Use existing AWS resources
+description: Configure the Amplify Libraries to use existing AWS resources by referencing them in your configuration.
```
Comment (brene) [inline docs/lib/analytics/existing-resources.md:3]: nit: "use existing AWS resources (Pinpoint, Kinesis)". Probably makes sense to have the actual service lists out inside the description. Might be helpful with SEO.

**Pull request merged**